src/Servers/HttpSys/samples/TestClient/Program.cs

```diff
@@ -11,20 +11,23 @@ namespace TestClient
     public class Program
     {
         private const string Address = 
-            // "http://localhost:5000/public/1kb.txt";
-            "https://localhost:9090/public/1kb.txt";
+            "http://localhost:5000/public/1kb.txt";
+            // "https://localhost:9090/public/1kb.txt";
 
         public static void Main(string[] args)
         {
-            WebRequestHandler handler = new WebRequestHandler();
-            handler.ServerCertificateValidationCallback = (_, __, ___, ____) => true;
+            Console.WriteLine("Ready");
+            Console.ReadKey();
+
+            var handler = new HttpClientHandler();
+            handler.MaxConnectionsPerServer = 500;
+            handler.ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator;
             // handler.UseDefaultCredentials = true;
-            handler.Credentials = new NetworkCredential(@"redmond\chrross", "passwird");
+            // handler.Credentials = new NetworkCredential(@"redmond\chrross", "passwird");
```

Comment:
Can we just remove the commented-out stuff from the sample?

Reply:
They're convenient for testing different scenarios, but I could at least separate out the scenarios to their own methods.

Reply:
If you want to test different scenarios, I liked what [SignalR did with making them an option you take in on the commandline/with startup args](https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/samples/ClientSample/Program.cs).